### PR TITLE
Remove questions and answers columns from events and applications tables

### PR DESCRIPTION
**Pull request by ariaru**
- Write and run migration
- Delete all code that references questions and answers

- Remember to run rake db:migrate

#84

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -71,7 +71,7 @@ def destroy
 
   private
     def event_params
-      params.require(:event).permit(:organizer_name, :organizer_email, :organizer_email_confirmation, :description, :name, :start_date, :end_date, :question_1, :question_2, :question_3, :approved, :ticket_funded, :accommodation_funded, :travel_funded, :deadline, :number_of_tickets, :website, :code_of_conduct, :city, :country)
+      params.require(:event).permit(:organizer_name, :organizer_email, :organizer_email_confirmation, :description, :name, :start_date, :end_date, :ticket_funded, :accommodation_funded, :travel_funded, :deadline, :number_of_tickets, :website, :code_of_conduct, :city, :country)
```
Comment (lisbethmarianne) [inline app/controllers/events_controller.rb]: Seems like you deleted :approved? Is this a mistake?

**Comment by lisbethmarianne** [comment]
Hi @ariaru, looks good to me. :+1:  
There is only one little problem I see in the
```
app/controllers/events_controller.rb
```
(I put a comment there.)

**Comment by ariaru** [comment]
Thanks for pointing that out. Changed it.

**Comment by ariaru** [comment]
Looks like the new commit is not automatically part of the old pull request. Not sure how I add it?

**Comment by lisbethmarianne** [comment]
I think it is, it says 2 commits.

**Comment by lisbethmarianne** [comment]
Seems all fine to me. Thanks for the work. Merging it. :heart:

**Pull request merged**